bin/npg_publish_tree.pl

```diff
@@ -220,6 +241,13 @@ sub handler {
   }
 }
 
+if ($stdio) {
+  my $metadata_in = _read_metadata_stdin();
+  foreach my $avu_hash (@{$metadata_in}) {
```

Comment:
Please don't append "types" to variable names. They don't impart much information and can give misleading information when code is changed.